src/api/images.tsx

```diff
@@ -16,13 +17,19 @@ export const getImage = async (id: string): Promise<APIImageWithURL> => {
   });
 };
 
+export const getImageQuery = (id: string, loader?: boolean) =>
```

Comment:
```suggestion
export const getImageQuery = (id: string, loader?: boolean) =>
```

Can you please rename the loader variable? It was specifically named this way in other functions to align with React Router loaders, but it should not be used in this case for clarity and to avoid confusion.